ci/android.groovy

```diff
@@ -26,7 +26,7 @@ def bundle() {
       )
     ]) {
       nix.shell(
-        "./gradlew assemble${target.capitalize()} ${gradleOpt}",
+        "./gradlew assembleRelease ${gradleOpt}",
```

Comment:
is it just for this PR or we will always build release builds now?

Reply:
just for PR

Reply:
so don’t forget to rollback these changes then